src/Nethermind/Nethermind.Cli/NodeManager.cs

```diff
@@ -38,9 +39,9 @@ public class NodeManager : INodeManager
 
         private readonly ICliConsole _cliConsole;
 
-        private JsonParser _jsonParser;
+        private readonly JsonParser _jsonParser;
```

Comment:
Is it reasonable to flag *all* private properties that are only ever read as `readonly`?

Reply:
most of them yes, there are perfomance implications both ways which are tricky